docs/README.md

```diff
@@ -180,6 +180,18 @@ module.exports = (webpackConfig, env) => {
 };
 ```
 
+### Output Filename
+
+Cosmos overrides `output.filename` value in webpack final config to `[name].js` by default. Some users are faced with caching `main.js` file. Use the `webpack.includeHashInOutputFilename` setting to change filename template to `[name].[contenthash].js`.
```

Comment:
This is good. I would add that users are faced with this caching issue when dealing with exports. In dev mod the filename is irrelevant, right?

Reply:
> ...with this caching issue when dealing with exports

Can you explain please?

> In dev mod the filename is irrelevant, right?

Yes.

Reply:
> Can you explain please?

I mean that this is an option specifically for the `cosmos-export` API. Not everybody generates static Cosmos exports, so I want to make sure people understand where this option is relevant.

Reply:
Ok, I mentioned about this in the readme: https://github.com/pavelivanov/react-cosmos/commit/6d919c6d760c05c575e243c9549da11aa3daa44c